tests/test_ext_autodoc.py

```diff
@@ -2024,6 +2025,151 @@ def test_pyclass_for_ClassLevelDocumenter(app):
     ]
 
 
+@pytest.mark.skipif(sys.version_info < (3, 6), reason='py36+ is available since python3.6.')
+@pytest.mark.sphinx('html', testroot='ext-autodoc')
+def test_pydata_for_DataDeclarationDocumenter(app):
```

Comment:
Is there any reason to test with all of data-declaration, py36-style typed attributes and py35-style typed attributes?